phpstan.neon.dist

```diff
@@ -1,13 +1,18 @@
-parameters:
-    paths:
-        - %currentWorkingDirectory%/
-    excludes_analyse:
-          - %currentWorkingDirectory%/vendor/php-stubs/wordpress-stubs/wordpress-stubs.php 
-    autoload_files:
-#        # Procedural code
-         - %currentWorkingDirectory%/util.php
-         #- %currentWorkingDirectory%/altapay.php
-         #stubs
-         - %currentWorkingDirectory%/stubs/wc_get_product.php
-    autoload_directories:
-         - %currentWorkingDirectory%/
+parameters:
+    paths:
+        - %currentWorkingDirectory%/
+    excludes_analyse:
+        - %currentWorkingDirectory%/vendor/
+        - %currentWorkingDirectory%/assets/
+    scanDirectories:
+        - ../woocommerce/
+        - ../../../wp-includes/
+        - ../../../wp-admin/
+#        - ../woocommerce-subscriptions
+#        - ../wordpress-develop-tests
+    level: 0
+    checkMissingIterableValueType: false
+    ignoreErrors:
+        - "#Inner named functions are not supported by PHPStan#"

```

Comment:
This can easily be solved by moving the 9 functions out of the outer functions.
```php
function init_altapay_settings() {
	function altapay_add_gateway( $methods ) {
	}
}
```
->
```php
function init_altapay_settings() {
}
function altapay_add_gateway( $methods ) {
}
```

Reply:
The issue is now fixed, but this line still needs to be removed from the config.